src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,84 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present